WordPress/Classes/Utility/InteractiveNotificationsManager.swift

```diff
@@ -687,3 +688,12 @@ extension InteractiveNotificationsManager: UNUserNotificationCenterDelegate {
         MeNavigationAction.notificationSettings.perform(router: UniversalLinkRouter.shared)
     }
 }
+
+private extension InteractiveNotificationsManager {
+    /// A temporary setting to allow controlling WordPress notifications when they are disabled after Jetpack installation
+    /// Disable WordPress notifications when they are enabled on Jetpack
+    func disableWordPressNotificationsIfNeeded() {
+        let notificationFilteringService = NotificationFilteringService()
```

Comment:
nit - perhaps we don't need to assign the local variable here anymore?

Reply:
I'm going to keep it as it is for now. 

I think it's more natural readability to have

`notificationFilteringService.disableWordPressNotificationsIfNeeded()` rather than `NotificationFilteringService().disableWordPressNotificationsIfNeeded()` although of course it's subjective and there're no clear guidelines regarding that.